packages/components/src/components/mobile-full-page-modal/mobile-full-page-modal.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classNames from 'classnames';
+import Div100vhContainer from '../div100vh-container';
+import FadeWrapper from '../fade-wrapper';
+import PageOverlay from '../page-overlay';
+
+const MobileFullPageModal = ({ children, className, header, is_modal_open, onClickClose, offset = '0px' }) => {
+    return (
+        <FadeWrapper
+            is_visible={is_modal_open}
+            className={classNames('mobile__full-page-modal ', `${className}__wrapper`)}
```

Comment:
Could we rename this to `dc-mobile-full-page-modal`? i.e. use `dc` prefixes and use correct BEM syntax (currently the name suggests `full-page-modal` is an element inside of `mobile`.